src/game_api/script/usertypes/vtables_lua.cpp

```diff
@@ -114,6 +114,9 @@ void register_usertypes(sol::state& lua)
         {
             Entity* ent = get_entity_ptr(uid);
             entity_vtable.unhook(ent, callback_id);
+            movable_vtable.unhook(ent, callback_id);
+            floor_vtable.unhook(ent, callback_id);
+            door_vtable.unhook(ent, callback_id);
```

Comment:
I don't understand this template jungle, but clear_callback didn't work and this fixed it.